charts/move2kube/values.yaml

```diff
@@ -1,6 +1,6 @@
 sshSecretName: sshkeys # name of the secret holding the ssh keys that will be used by move2kube resources
-brokerName: default # name of the broker used by Knative eventing resources
-brokerURL: http://broker-ingress.knative-eventing.svc.cluster.local/sonataflow-infra/default
+brokerName: kafka-broker # name of the broker used by Knative eventing resources
+brokerURL: http://kafka-broker-ingress.knative-eventing.svc.cluster.local/orchestrator/kafka-broker
```

Comment:
where is this brokerURL being used?

Reply:
ah, you are right, this is not used anymore, I even though I deleted it few months back, thanks for catching it